tests/test.py

```diff
@@ -0,0 +1,31 @@
+import pybamm
+import pybamm_tea as TEA
+import matplotlib.pyplot as plt
+import matplotlib.patches as patches
+import pandas as pd
+
+# load example model
```

Comment:
you should use something like `unittest` or `pytest` to set up testing. E.g. look [here](https://github.com/pybamm-team/BPX/blob/main/tests/test_utilities.py) for an example using `unittest`. You need to add test classes that inherit from `unittest.TestCase` and define the tests in methods.